test/functional/apps/dashboard_elements/controls/common/control_group_chaining.ts

```diff
@@ -152,18 +152,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       await dashboardControls.ensureAvailableOptionsEqual(controlIds[1], {
         suggestions: { Fluffy: 6, 'Fee Fee': 3, Rover: 3 },
-        invalidSelections: ['sylvester'],
+        invalidSelections: ['Invalid selection.\nsylvester'],
```

Comment:
This changed because the `EuiIcon` adds the text where we used `EuiScreenReader` before. See https://github.com/elastic/kibana/pull/174201/files#r1473370193